components/commitdiff/commitdiff.html

```diff
@@ -1,17 +1,19 @@
-<div class="btn-group btn-group-xs" data-bind="visible: showDiffButtons">
-  <button class="btn btn-default bootstrap-tooltip pull-right commit-whitespace" data-bind="click: whiteSpace.toggle, css: {active: wordWrap.isActive}" data-toggle="tooltip" data-placement="bottom" title="Ignore whitespaces" data-delay='{"show":"2000", "hide":"0"}'>
-    <span data-bind="text: whiteSpace.text"></span>
-  </button>
-</div>
-<div class="btn-group btn-group-xs" data-bind="visible: showDiffButtons">
-  <button class="btn btn-default bootstrap-tooltip pull-right commit-sideBySideDiff" data-bind="click: textDiffType.toggle, css: {active: textDiffType.isActive}" data-toggle="tooltip" data-placement="bottom" title="Show side by side diff view" data-delay='{"show":"2000", "hide":"0"}'>
-    <span data-bind="text: textDiffType.text"></span>
-  </button>
-</div>
-<div class="btn-group btn-group-xs" data-bind="visible: showDiffButtons">
-  <button class="btn btn-default bootstrap-tooltip pull-right commit-wordwrap" data-bind="click: wordWrap.toggle, css: {active: wordWrap.isActive}" data-toggle="tooltip" data-placement="bottom" title="Wrap words per line" data-delay='{"show":"2000", "hide":"0"}'>
-    <span data-bind="text: wordWrap.text"></span>
-  </button>
+<div class="btn-toolbar">
```

Comment:
`showDiffButtons` binding can be added here instead of each of the `btn-group`s.

```suggestion
<div class="btn-toolbar" data-bind="visible: showDiffButtons">
```

Reply:
Thanks, fixed in f799d90.